hw5/hw5.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from pyspark.sql import SparkSession
+
+# Create a spark session
+spark = (
+    SparkSession.builder.config(
+        "spark.jars",
+        "/Users/zack/Documents/SDSU/Fall 2022/mysql-connector-java-5.1.46/mysql-connector-java-5.1.46.jar",
+    )
+    .master("local")
+    .appName("HW5")
+    .getOrCreate()
+)
+
```

Comment:
Not sure if you need to use spark to load the your data table in. I ran your SQL script and you have around 15,800 which is manageable using pythons pandas and sql alchemy. SQLalchemy might save you some steps in the future